code/modules/projectiles/guns/ballistic/automatic.dm

```diff
@@ -344,3 +344,71 @@
 	actions_types = list()
 	fire_sound = 'sound/weapons/laser.ogg'
 	casing_ejector = FALSE
+
+// AWP //
+
+/obj/item/gun/ballistic/automatic/sniper_rifle/alt
+	name = "AWP"
+	desc = "A powerful sniper rifle chambered in .338 lapua magnum. It kinda smells like tears of silvers, whatever that means."
+	icon_state = "sniper_alt"
+	item_state = "sniper_alt"
+	fire_sound = 'sound/weapons/awp_fire.ogg'
+	fire_sound_volume = 100
+	vary_fire_sound = FALSE
+	load_sound = 'sound/weapons/awp_clipin.ogg'
+	rack_sound = 'sound/weapons/boltpull.ogg'
+	recoil = 2
+	weapon_weight = WEAPON_HEAVY
+	mag_type = /obj/item/ammo_box/magazine/sniper_rounds/alt
+	fire_delay = 40
+	burst_size = 1
+	w_class = WEIGHT_CLASS_NORMAL
+	zoomable = TRUE
+	zoom_amt = 10
+	zoom_out_amt = 13
+	slot_flags = ITEM_SLOT_BACK
+	actions_types = list()
+	mag_display = TRUE
+	bolt_type = BOLT_TYPE_STANDARD
```

Comment:
```suggestion
```
Redundant.
You already define this as BOLT_TYPE_STANDARD from your ballistic parent.